js/components/App.js

```diff
@@ -40,6 +40,7 @@ function select(state) {
   return {
     messages: filteredMessages.messages,
     filterText: state.filters.filterText,
+    filterSeverity: state.filters.filterSeverity,
```

Comment:
In the final version, we'll want to look to see which props are only used in children. For example, if `filterText/filterSeverity` is only used in `ConsoleToolbar`, we might take it out of this selector and use connect in `ConsoleToolbar`.